truffle-config.js

```diff
@@ -34,15 +34,19 @@ module.exports = {
         process.env.MAINNET_MNEMONIC, 'https://mainnet.infura.io', 0, 2,
       ),
       network_id: '1',
-      gas: 200000,
+      // hard gas limit
+      gas: 6000000,
       gasPrice: 5000000000 // Specified in Wei
     },
     rinkeby: {
       provider: () => new HDWalletProvider(
         process.env.RINKEBY_MNEMONIC, 'https://rinkeby.infura.io', 0, 2,
       ),
       network_id: '4',
-      gas: 200000
+      // hard gas limit
+      gas: 6000000,
```

Comment:
Isn't having such a limit here makes all the individual estimates not necessary?

Reply:
truffle complaints if these are not there. Maybe a safety measure 🤷‍♂️

Reply:
What's the complain?

To illustrate my point: https://github.com/leapdao/leap-contracts/pull/157/
This works for me for ganache and rinkeby without any hardcoded estimates.

Reduced gas limit to 5.5m because we don't have larger txs

Reply:
@troggy Course it works, because that PR will set a gas limit to 5.5 mil for every transaction.
@johannbarbie mentioned that this will take longer to mine tho. 🤷‍♂️ 
I don't have experience on this part 🙂